pkg/utils/status.go

```diff
@@ -388,7 +403,7 @@ func calculateState(ctx context.Context, instance *csmv1.ContainerStorageModule,
 	newStatus.State = constants.Succeeded
 	// TODO: Currently commented this block of code as the API used to get the latest deployment status is not working as expected
```

Comment:
I guess I'm ok with getting rid of this code, since we are planning on refactoring status anyway, so we can check on controller-runtime then. Thoughts @JacobGros ?

Reply:
makes sense to me
if we ever decide we need that code again, we can look at an older operator version